software/runtime/data/data_cfft_radix4_q16.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
```

Comment:
Those Python scripts and templates are very similar and could likely easily merge into one. But we can tackle this while reworking the data generation in general.

Reply:
Agreed.